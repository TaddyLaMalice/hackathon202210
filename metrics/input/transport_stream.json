{
	
	"transformers": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/home/outscale/media/test_transformers.ts",
			"md5sum": "todo"
		}
	},
	"scrat": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/home/outscale/media/test_scrat.ts",
			"md5sum": "todo"
		}
	},
	"casino_royale": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/home/outscale/media/test_casino_royale.ts",
			"md5sum": "todo"
		}
	}
}
