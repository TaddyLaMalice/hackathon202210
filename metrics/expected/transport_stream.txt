transformers [{'pid': 482, 'deltaPcrPts': {'min': 212.69040740740456, 'max': 316.8268518518453, 'average': 262.9774161943686}, 'duration': 71.936, 'size': 1785060, 'bandwidth': 198516.4590747331}, {'pid': 481, 'deltaPcrPts': {'min': 348.06181481482054, 'max': 992.3344074074121, 'average': 715.6008758497151}, 'duration': 72.0, 'size': 64571232, 'bandwidth': 7174581.333333333}]
scrat [{'pid': 4113, 'deltaPcrPts': {'min': 390.34444444440305, 'max': 495.9222222222015, 'average': 449.6829124187921}, 'duration': 430.24, 'size': 370233288, 'bandwidth': 6884218.817404239}, {'pid': 4352, 'deltaPcrPts': {'min': 367.11111111100763, 'max': 440.766666666721, 'average': 402.1453407867095}, 'duration': 452.7426777777779, 'size': 10615984, 'bandwidth': 187585.30213421938}]
casino_royale [{'pid': 48, 'deltaPcrPts': {'min': 121.08744444444892, 'max': 1239.722222222248, 'average': 858.9120531966983}, 'duration': 895.6, 'size': 574721076, 'bandwidth': 5133730.022331398}, {'pid': 64, 'deltaPcrPts': {'min': 4.010814814828336, 'max': 33.73185185180046, 'average': 19.009071679118087}, 'duration': 894.72, 'size': 51393560, 'bandwidth': 459527.539341917}, {'pid': 65, 'deltaPcrPts': {'min': 3.579407407407416, 'max': 33.45192592591047, 'average': 18.897631015206674}, 'duration': 894.72, 'size': 51393560, 'bandwidth': 459527.539341917}, {'pid': 80, 'deltaPcrPts': {'min': 1805.1082592592575, 'max': 2099.0876296296337, 'average': 2011.9996780045378}, 'duration': 541.4, 'size': 1690308, 'bandwidth': 24976.84521610639}]
